readme.md

```diff
@@ -18,14 +18,15 @@ PHP 5.6 or higher.
 Installation
 ------------
 
-The best way to install Web Project is using Composer. If you don't have Composer yet,
-download it following [the instructions](https://doc.nette.org/composer). Then use command:
+The best way to install this Web Project is by using Composer. If you don't have Composer yet,
+download it by following [these instructions](https://doc.nette.org/composer). Then use this 
+command in your console:
 
 	composer create-project nette/web-project path/to/install
 	cd path/to/install
 
 
-Make directories `temp/` and `log/` writable.
+Disable read-only for the `temp/` and `log/` directories.
```

Comment:
Well, the first word `disable` looks strange. I get your point, but `disable read-only` is confusing.

Do you have any other ideas? Such as `Make writable these folders: temp and log.

Reply:
> What is wrong with "make writable"?

It seems good to me.